Runtime/Scripts/BaseMeshSync.cs

```diff
@@ -1349,9 +1349,34 @@ private static bool HandleKeywords(Material destMat, string keyword)
                 break;
             case IMaterialPropertyData.Type.Vector:
                 if (len == 1)
-                    destMat.SetVector(propNameID, prop.vectorValue);
+                {
+                    // Apply gamma correction if needed:
+                    Color propertyValueAsColor = prop.vectorValue;
+                    var col = QualitySettings.activeColorSpace == ColorSpace.Linear ? propertyValueAsColor.gamma : propertyValueAsColor;
```

Comment:
This seems to be the other way around ?
Can you pass me a sample to test this ?
Also, is this because the color space on the DCC side is not linear ?

Reply:
Do other DCC tools use gamma? Might be good to pass the color space the dcc tool uses.

Reply:
> Do other DCC tools use gamma? Might be good to pass the color space the dcc tool uses.

Yeah, I second this opinion.

Reply:
I've made a change to only do this for blender now.

Reply:
Is it possible to use linear space in Blender too ?

Reply:
This handles just the default blender display device setting of sRGB and standard view transform. Different settings for these will change the colours and it won't match unity anymore. Maybe we should add a setting on the server instead?

Reply:
I've changed it to use the dcc settings now as discussed. Related PR here: https://github.com/Unity-Technologies/MeshSyncDCCPlugins/pull/386/files